src/pages/getting-started/app-guides/integrations/index.mdx

```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
+  
+</div>
+
+## Link deployment to a chosen service
+
+Now, you need to link deployment to one of the available deployment services, Netlify or Vercel. This step requires connecting your Developer Program account with your Netlify or Vercel profile. You'll be redirected to a chosen provider's website to authorize this operation. This permission allows your Developer Console application to create and manage sites (or projects if we refer to Vercel) in your Netlify or Vercel profile.  
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment" src="/images/getting-started/app-guides/integrations/livechat-link-deployment-new-site.png" width="1406px" height="750px"/>
+  
+</div>
+
+💡 If you have more than one account or repository on your Netlify or Vercel account, you can choose which account and repository you want to give the permissions to.
+
+## Connect your deployment service to your repo
+
+When the connection is successful, you can choose whether to **Create a new site/project** or **Use an existing site/project**. This operation aims at connecting your Netlify or Vercel account to your repository.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment Netlify" src="/images/getting-started/app-guides/integrations/livechat-link-deployment.png" width="1406px" height="750px"/>
+  
+</div>
+
+If you use Netlify, click the **Link** button after creating your site or using the existing one. When the deployment in Netlify is created, it's time to connect your Netlify account to your app's repository. To do that, click the **Install** button in the **Install the Netlify GitHub app**.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment Netlify permission" src="/images/getting-started/app-guides/integrations/livechat-deployment-almost-ready.png" width="1406px" height="750px"/>
+  
+</div>
+
+If you use Vercel, you need to permit Vercel to access your repository before creating your project or using the existing one. You’ll be redirected to Vercel's website to authorize this connection. When it’s done, you can link your deployment.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment Vercel" src="/images/getting-started/app-guides/integrations/livechat-vercel-app-permission.png" width="1406px" height="750px"/>
+  
+</div>
+
+## Check what Netlify or Vercel integration gives you
+
+Your Deployment block is all set, and your app URL is available now. You have your Developer Console connected to the GitHub repository and to the Netlify or Vercel profile. Your GitHub account is also connected to the selected provider's account. You can go to the created site/project in Netlify or Vercel or open your app directly from the Console. You can also use the provided URL to add more building blocks to your app.
```

Comment:
```suggestion
By now your Deployment should be all set, and your app URL available to access! You can open your application's deployment directory directly from the Console, but you can also find your created site (or project) in the deployment service you integrated with. You can also use the provided URL to add more building blocks to your app.
```

Reply:
this section sounded a little off to me 😆 I made some "big" changes but feel free to rephrase as you see fit if you don't like it

Reply:
I totally agree with you now that you have written your opinion.